qdrant-landing/content/documentation/concepts/points.md

```diff
@@ -1632,24 +1631,144 @@ All resulting points are sorted by ID. To query the next page it is necessary to
 For convenience, this ID is also returned in the field `next_page_offset`.
 If the value of the `next_page_offset` field is `null` - the last page is reached.
 
-<!-- 
-Python client:
+### Order points by a payload key
+
+_Available as of v1.8_
```

Comment:
We did specify patch version for older releases, e.g.
https://github.com/qdrant/landing_page/blob/069857d0c307abfaba8e4a3ae4a300c1c2004e11/qdrant-landing/content/documentation/concepts/points.md#L818


```suggestion
_Available as of v1.8.0_
```